aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/DeleteHandler.java

```diff
@@ -39,23 +40,22 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
             final AmazonWebServicesClientProxy proxy,
             final ResourceHandlerRequest<ResourceModel> request,
             final CallbackContext callbackContext,
-            final ProxyClient<RdsClient> proxyClient,
-            final Logger logger
-    ) {
+            final ProxyClient<RdsClient> rdsProxyClient,
+            final ProxyClient<Ec2Client> ec2ProxyClient, final Logger logger) {
```

Comment:
nit: can you move logger to the newline :)

Reply:
Done.